Text3DGeometry example does not load

Visited: https://threlte.xyz/docs/reference/extras/text-3d-geometry/

The example is not displayed and there are errors in the console:

```raycast.CxKKhBrm.js:1 Error in asyncWritable: Failed to fetch
(anonymous) @ raycast.CxKKhBrm.js:1Understand this error
observe.svelte.BHiIsSLZ.js:44  

Uncaught (in promise) TypeError: Failed to fetch
    at qn.load (observe.svelte.BHiIsSLZ.js:44:128899)
    at O.load (observe.svelte.BHiIsSLZ.js:44:133150)
    at O.load (RGBELoader.B8kJQCHQ.js:5:2176)
    at observe.svelte.BHiIsSLZ.js:44:127929
    at new Promise (<anonymous>)
    at O.loadAsync (observe.svelte.BHiIsSLZ.js:44:127901)
    at Environment.tb-KOlva.js:6:26733
    at Object.remember (observe.svelte.BHiIsSLZ.js:1:1355)
    at Environment.tb-KOlva.js:6:26714
    at Bt (runtime.D20saW8m.js:1:20747)```

<img width="1050" height="1204" alt="Image" src="https://github.com/user-attachments/assets/a847f8f2-9680-4c54-8e04-a68129e3fcbc" />

--- a/.changeset/flat-rules-occur.md
+++ b/.changeset/flat-rules-occur.md
@@ -1,0 +1,5 @@
+---
+"@threlte/extras": patch
+---
+
+Fix Align component stopping async resource loading in child components

--- a/packages/extras/src/lib/components/Align/Align.svelte
+++ b/packages/extras/src/lib/components/Align/Align.svelte
@@ -79,7 +79,7 @@
   useTask(
     () => {
       calculate()
-      stop()
+      scheduleAligning = false
     },
     { stage, running: () => scheduleAligning }
   )

